### DISCUSSION TITLE
error permisiion

### DISCUSSION THREAD

**Discussion by msyamsul725** [codemagic-ci-cd/codemagic-docs#1278] [Q&A] [answered]
HTTP Error: 403, The caller does not have permission .

**Comment by K-Codemagic** [comment, marked as answer]
Hi, are you receiving this error while publishing to Google Play, if so, please check [this](https://docs.codemagic.io/troubleshooting/common-google-play-errors/#code-403-the-caller-does-not-have-permission-status-permission_denied) out.

**Reply by msyamsul725** [reply]
I release for firebase mr

**Reply by K-Codemagic** [reply]
Can you please share the build id? However, as per the error, it seems to be a permission issue.

**Reply by msyamsul725** [reply]
yes sir, this is my first time using code magic, so I'm having a little trouble

**Reply by msyamsul725** [reply]
Publishing failed :| Failed to publish /Users/builder/clone/build/app/outputs/flutter-apk/app-release.apk to Firebase App Distribution: [1m[36mi [39m[22m uploading binary... [1m[31mError:[39m[22m failed to upload release. HTTP Error: 403, The caller does not have permission

**Reply by K-Codemagic** [reply]
Can you please check if your service account role is correct? Please check [this](https://docs.codemagic.io/flutter-publishing/firebase-app-distribution/) guide and let me know how it goes.

**Reply by K-Codemagic** [reply]
Hi, is this issue resolved?

**Comment by msyamsul725** [comment]
Solved sir , thank you

Pada tanggal Rab, 18 Jan 2023 18.36, K-Codemagic ***@***.***>
menulis:
